tests/toolchain/gcc5_fortran_compilation.pm

```diff
@@ -9,7 +9,7 @@
 # without any warranty.
 
 # G-Summary: Fortran test for SLE Toolchain Module's GCC5
-# G-Maintainer: Michal Nowak <mnowak@suse.com>
+# G-Maintainer: Michal Nowak <mnowak@suse.com>, Michal Nowak <mnowak@suse.com>
```

Comment:
you *and* your twin? Please remove the G prefixes and just have one maintainer. it doesn't have to be perfect